product_replenishment_cost/models/product_template.py

```diff
@@ -11,12 +11,23 @@
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
-    # TODO this field should be move to PRC (product_replenishment_cost)
+    replenishment_cost = fields.Float(
+        'Replenishment Cost',
+        compute='_compute_replenishment_cost',
+        # TODO, activamos store como estaba??
+        store=False,
+        digits=dp.get_precision('Product Price'),
+        help="The cost that you have to support in order to produce or "
+             "acquire the goods. Depending on the modules installed, "
+             "this cost may be computed based on various pieces of "
+             "information, for example Bills of Materials or latest "
+             "Purchases. By default, the Replenishment cost is the same "
+             "as the Cost Price.")
+
     replenishment_cost_last_update = fields.Datetime(
         'Replenishment Cost Last Update',
         track_visibility='onchange',
     )
-    # TODO this field should be move to PRC (product_replenishment_cost)
     replenishment_base_cost = fields.Float(
         'Replenishment Base Cost',
```

Comment:
>      replenishment_base_cost = fields.Float(
>          'Replenishment Base Cost',

same here.
remove this field string since the string value is the same of the field name redundant